mobile/src/bundles/talent/components/skills-and-projects-form/not-consider-types.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+
+import { Checkbox, View } from '~/bundles/common/components/components';
+import { splitArrayInHalf } from '~/bundles/common/helpers/helpers';
+import { useCallback, useFormController } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties<T extends FieldValues> = {
+    name: FieldPath<T>;
+    control: Control<T, null>;
+    options: string[];
+};
+
+const NotConsiderTypes = <T extends FieldValues>({
+    name,
+    control,
+    options,
+}: Properties<T>): JSX.Element => {
+    const { field } = useFormController({ name, control });
+    const { value, onChange } = field;
+
+    const [column1Options, column2Options] = splitArrayInHalf(options);
```

Comment:
In this component there are lots off repeatings from other code, let's create a separate component for this column options